src/main/java/org/zeromq/ZMQ.java

```diff
@@ -3637,11 +3633,8 @@ protected Poller(Context context, int size)
             selector = context.selector();
             assert (selector != null);
 
-            items = new PollItem[size];
+            items = new LinkedList<>();
```

Comment:
As size is now useless, the constructor using a size should be deprecated. Or size could be used when creating the list, it avoid some useless resizing.

Reply:
resizing is cheap on LinkedList, but you're right, I should deprecate this constructor and create a new one without size